types/guacamole-client/lib/Keyboard.d.ts

```diff
@@ -52,7 +52,7 @@ export class Keyboard {
      * Element must be manually provided through the listenTo() function for
      * the Guacamole.Keyboard instance to have any effect.
      */
-    constructor(element: HTMLDocument | HTMLElement);
+    constructor(element?: HTMLDocument | HTMLElement);
```

Comment:
https://github.com/apache/guacamole-client/blob/e550b244f8bfefdbf26aeed6b5bba7c78e43e6d9/guacamole-common-js/src/main/webapp/modules/Keyboard.js#L1434 The element is optional